shopcart-backend/src/main/java/com/shopcart/backend/entity/Order.java

```diff
@@ -0,0 +1,103 @@
+package com.shopcart.backend.entity;
+
+import java.sql.Date;
+import java.util.HashMap;
+import java.util.Map;
+
+import javax.persistence.CollectionTable;
+import javax.persistence.Column;
+import javax.persistence.ElementCollection;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.MapKeyColumn;
+import javax.persistence.Table;
+import javax.validation.constraints.NotBlank;
+
+import org.hibernate.annotations.CreationTimestamp;
+
+@Entity
+@Table(name="orders")
+public class Order {
```

Comment:
We need an order status field where the statuses are an Enumeration such as PROCESSING, PAID, etc.

Reply:
@ReubenMathew in this case what purpose would this serve and how would it be done? Because if when the user sends the post request to pay for an order it processes the payment at the same time, meaning there's never a moment where an order would be in a "PROCESSING" state as either it is "PAID" or its denied and no order is made

Reply:
Have added an enum but the PROCESSING status is still TBD on when we actually will have an order use it

Reply:
Added processing endpoint as well as item quantity checks and a submit endpoint to process payment and deduct stock